module/zfs/spa_stats.c

```diff
@@ -869,8 +870,42 @@ spa_mmp_history_destroy(spa_t *spa)
 	mutex_destroy(&ssh->lock);
 }
 
+/*
+ * Set duration in existing "skip" record to how long we have waited for a leaf
+ * vdev to become available.
+ *
+ * Important that we start search at the end of the list where new records are
```

Comment:
s/at the end of the list/at the head of the list/

Reply:
fixed